magister-calendar.js

```diff
@@ -311,7 +317,7 @@ function parseAppointments(appointments, currentcourse) {
     });
   }
 
-  /* The following block of code is written for a specific school using Magister,
+  /* The following block of code is written for two specific schools using Magister,
```

Comment:
Alright. So this special handling of appointment start and end times is starting to get a little out-of-scope of the main functionality of the program and 'bloated'. 

I suggest to move these blocks to separate modules, for example in a `config/appointment-time-adjustments/{schoolname}.js`. And load the appropriate modules here based on the Magister school.

What do you think?